### DISCUSSION TITLE
Build an ios app from windows with the Microsoft.Appcenter nuget crash on run

### DISCUSSION THREAD

**Discussion by softlion** [dotnet/maui#9027] [General] [answered]
Hey :)
I tried to start my app on an iOS simulator.
It is a maui android+ios app, and i'm using VS Preview on windows.

The app build ok, it starts, but it crashes directly on an AppCenter call.

Any clue ?

```
System.TypeInitializationException: The type initializer for 'Microsoft.AppCenter.Utils.ApplicationLifecycleHelperDesktop' threw an exception.
 ---> System.IO.FileNotFoundException: Could not load file or assembly 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
File name: 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
   --- End of inner exception stack trace ---
   at Microsoft.AppCenter.ApplicationLifecycleHelper.get_Instance()
   at Microsoft.AppCenter.AppCenter..ctor()
   at Microsoft.AppCenter.AppCenter.get_Instance()
   at Microsoft.AppCenter.AppCenter.PlatformStart(String appSecret, Type[] services)
   at Microsoft.AppCenter.AppCenter.Start(String appSecret, Type[] services)
   ...
```

**Comment by batespe** [comment]
> 

I have exactly the same issue, anyone find a fix?

**Comment by madmapper** [comment]
Same issue - app crashes when trying to start up App Center

**Reply by madmapper** [reply]
Sorry hit send too soon.  Having the same issue when building and running using Visual Studio Mac

**Comment by Taiizor** [comment]
It gives me the same error when compiling an application from Windows to iOS (MAUI).

System.TypeInitializationException: The type initializer for 'Microsoft.AppCenter.Utils.ApplicationLifecycleHelperDesktop' threw an exception.
 ---> System.IO.FileNotFoundException: Could not load file or assembly 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' or one of its dependencies.
File name: 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
   --- End of inner exception stack trace ---
   at Microsoft.AppCenter.ApplicationLifecycleHelper.get_Instance()
   at Microsoft.AppCenter.AppCenter..ctor()
   at Microsoft.AppCenter.AppCenter.get_Instance()
   at Microsoft.AppCenter.AppCenterService.get_ApplicationSettings()
   at Microsoft.AppCenter.AppCenterService.get_InstanceEnabled()
   at Microsoft.AppCenter.Crashes.Crashes.get_InstanceEnabled()

**Comment by awalker-dsg** [comment]
We saw the same exception when starting iOS on MAUI using the Microsoft.AppCenter v4.5.3 NuGet. Updating to v5.0.1 resolved the issue.

**Reply by batespe** [reply]
Yes, this works for me as well. I think we can consider this fixed.

**Comment by softlion** [comment, marked as answer]
Fixed !
